projects/rero/ng-core/src/lib/record/detail/detail.component.html

```diff
@@ -17,7 +17,15 @@
 <div class="main-content">
   <ng-core-error [error]="error" *ngIf="error"></ng-core-error>
   <div class="float-right ml-4 mt-2 mb-4" *ngIf="record && adminMode.can">
-    <a href class="btn btn-sm btn-primary" [routerLink]="['../../edit', record.metadata.pid]"
+    <button *ngIf="useStatus && useStatus.can && useStatus.url"
+            id="detail-use-button"
+            class="btn btn-sm btn-outline-primary ml-1"
```

Comment:
For the templates, I would not put this button in outline, as it is the primary action. However, the edit button should be outline in this particular case.

Reply:
done